src/components/ResponsiveList/ResponsiveList.jsx

```diff
@@ -14,20 +14,28 @@ const ResponsiveList = forwardRef(
     {
       id,
       className,
+      rootClassName,
       children,
       menuButtonSize,
       paddingSize,
       dialogZIndex,
       dialogClassName,
       menuButtonClassName,
       resizeDebounceTime,
+      menuButtonAriaLabel,
       menuButtonProps
     },
     ref
   ) => {
     const componentRef = useRef(null);
     const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
-    const index = useElementsOverflowingIndex({ ref: componentRef, children, paddingSize, resizeDebounceTime });
+    const index = useElementsOverflowingIndex({
```

Comment:
From what I saw we're the only one using ResponsiveList and useElementsOverflowingIndex
so there's no risk of breaking changes right?

Reply:
it is open source, so it not known, but this is still a work in progress, i don't mind changing it